api/rpc/project/project.proto

```diff
@@ -2,15 +2,30 @@ syntax = "proto3";
 
 package project;
 
-service Project {
-  rpc Create (CreateRequest) returns (CreateReply) {}
+service ProjectSvc {
```

Comment:
In order to be consistent with other proto files, I would name the service "Project".